.github/workflows/functional-test.yaml

```diff
@@ -575,6 +577,15 @@ jobs:
           BICEP_RECIPE_REGISTRY: ${{ env.BICEP_RECIPE_REGISTRY }}
           BICEP_RECIPE_TAG_VERSION: ${{ env.BICEP_RECIPE_TAG_VERSION }}
           GH_TOKEN: ${{ steps.get_installation_token.outputs.token }}
+          GOTESTSUM_OPTS: '--junitfile ./dist/functional_test/results.xml'
+      - name: Process Functional Test Results
+        uses: ./.github/actions/process-test-results
+        # In case of failure, upload functional_test_results to artifacts so that they are not erased by subsequent runs.
+        if: failure() && github.repository == 'radius-project/radius'
+        with:
+          test_group_name: 'Functional Tests'
+          artifact_name: 'functional_test_results'
```

Comment:
These names need to be unique using the matrix name

Reply:
fixed